test/remoting.integration.js

```diff
@@ -118,13 +118,14 @@ describe('remoting - integration', function() {
         'find(filter:object):store GET /stores',
         'findOne(filter:object):store GET /stores/findOne',
         'updateAll(where:object,data:object) POST /stores/update',
-        'deleteById(id:any) DELETE /stores/:id',
+        'deleteById(id:any):number DELETE /stores/:id',
```

Comment:
I assume I need to change the memory connector impl to return boolean instead of object with a count property. This will probably break a bunch of tests on the way, so then I will have to fix all those down the chain too I assume.

@chandadharap Increasing scope for this story. FYI.